skrub/_expressions/_skrub_namespace.py

```diff
@@ -2002,7 +2002,7 @@ def is_X(self):
 
     @check_expr
     def mark_as_y(self):
-        """Mark this expression as being the ``X`` table.
+        """Mark this expression as being the ``y`` vector.
```

Comment:
Hi @emilienbattel09, thank you for fixing this typo! `y` can be a table (for instance `RandomForestClassifier` supports multi-label classes)
```suggestion
        """Mark this expression as being the ``y`` table.
```